drivers/adc-dac/ad5592r/ad5593r.c

```diff
@@ -144,7 +145,13 @@ int32_t ad5593r_reg_write(struct ad5592r_dev *dev, uint8_t reg,
 	data[1] = value >> 8;
 	data[2] = value;
 
-	return i2c_write(dev->i2c, data,sizeof(data), STOP_BIT);
+	ret = i2c_write(dev->i2c, data,sizeof(data), STOP_BIT);
+
+	if (reg == AD5592R_REG_RESET && ret < 0) {
```

Comment:
Is this `ret < 0` indented ? I tried to find in the datasheet but failed to find how the chip is supposed to behave when a software reset is issued. If this condition is empyrical I think we should remove it.

Reply:
It is intended, if the check fails but a reset is what is being performed the error should be ignored. I have attached an excerpt from the datasheet (pg.30 - AD5593R). 

![ad5593r_reset](https://user-images.githubusercontent.com/56347769/82819568-7bf39500-9e98-11ea-9af6-80e4bf113fb0.png)

Reply:
exactly what i was looking for but missed it, thanks